core/commands/wallet.go

```diff
@@ -52,24 +57,81 @@ withdraw and query balance of token used in BTFS.`,
 		"transfer":          walletTransferCmd,
 		"discovery":         walletDiscoveryCmd,
 		"validate_password": walletCheckPasswordCmd,
+		"generate_key":      walletGenerateKeyCmd,
 	},
 }
 
 var walletInitCmd = &cmds.Command{
 	Helptext: cmds.HelpText{
-		Tagline:          "Init BTFS wallet",
-		ShortDescription: "Init BTFS wallet.",
+		Tagline:          "initialize BTFS wallet",
+		ShortDescription: "initialize BTFS wallet",
+	},
+	Arguments: []cmds.Argument{
+		cmds.StringArg("privateKey", true, false, "private key"),
```

Comment:
use `private_key`

Reply:
i thought it's a no side effects api

Reply:
oh but for consistency we  should use "_" ?

Reply:
I think the problem is consistency, it is always `privateKey` according to `privateKeyOptionName` (should just use this variable), but the rest of the new arguments are in "_" format. Should modify to use a single format (probably stick with privateKey). @laipogo

Reply:
ah, sorry, i comments in a wrong place...
actually i prefer snake style.
it was a typo here.

Reply:
it's fixed.